builtin/providers/aws/resource_aws_rds_cluster_instance.go

```diff
@@ -83,6 +83,20 @@ func resourceAwsRDSClusterInstance() *schema.Resource {
 				Computed: true,
 			},
 
+			"kms_key_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
```

Comment:
Ah sorry, I posted another below that https://github.com/hashicorp/terraform/issues/7655#issuecomment-232952498